articles/index.md

```diff
@@ -12,18 +12,18 @@ and integration that was not possible with the previous JavaScript language serv
 
 > [!WARNING]
 > With the new JavaScript language service, while many new features were enabled, some features
-> were also removed. In particular, [intellisense extensions](https://msdn.microsoft.com/en-us/library/hh874692.aspx)
+> were also removed. In particular, [IntelliSense extensions](https://msdn.microsoft.com/en-us/library/hh874692.aspx)
```

Comment:
This isn't the only file where the capitalization of "IntelliSense" is incorrect. 😄